ci/doc/how-to-install-taos-ci.md

```diff
@@ -40,10 +40,10 @@ $ git clone https://github.com/<your_account>/<your_prj_name>.git <your_prj_name
 $ cd <your_prj_name>.git
 $ git submodule add https://github.com/<your_account/TAOS-CI.git
 $ ln -s ./TAOS-CI/ci ./ci
-$ vi ./ci/taos/config/config-cibot.sh
-  (Please modify configuration variables appropriately.)
+$ vi ./ci/taos/config/config-webhook.sh
```

Comment:
config-webhook.sh -> config-webhook.json

Reply:
No. it right. it is a json file.